src/components/LoginForm/Login.tsx

```diff
@@ -0,0 +1,65 @@
+"use client";
+
+import { useState } from "react";
+
+import FormFieldsHeader from "@/components/LoginForm/FormFieldsHeader";
+import PersonalFormFields from "@/components/LoginForm/PersonalFormFields";
+import type { AuthenticatorProps } from "@aws-amplify/ui-react";
+import {
+  Authenticator,
+  Button,
+  View,
+  useAuthenticator,
+} from "@aws-amplify/ui-react";
+import type { DefaultComponents } from "@aws-amplify/ui-react/dist/types/components/Authenticator/hooks/useCustomComponents/defaultComponents";
+import { type AuthContext } from "@aws-amplify/ui/dist/types";
+
+export default function Login() {
+  const components: DefaultComponents = {
+    SignUp: {
+      Header: () => <FormFieldsHeader />,
+    },
+    SignIn: {
+      Header: () => <FormFieldsHeader />,
+    },
+    Footer: () => {
+      const { toSignIn, toSignUp } = useAuthenticator();
+      const [signIn, setSignIn] = useState(false);
+      return (
+        <View textAlign="center">
+          <Button
+            fontWeight="normal"
+            onClick={() => {
+              signIn ? toSignIn() : toSignUp();
+              setSignIn(!signIn);
+            }}
+            size="small"
+          >
+            {signIn
+              ? "Already have an account?"
+              : "Don't have an account? Sign up"}
+          </Button>
+        </View>
+      );
+    },
+  };
+
+  const services: AuthContext["services"] = {};
+  const formFields: AuthenticatorProps["formFields"] = {
+    signUp: {},
+  };
+  return (
+    <Authenticator
+      socialProviders={["google", "apple"]}
+      initialState="signIn"
```

Comment:
Could we make it so that this is based on if we're on the /login path or the /register path?  useRouter() and query.path might be useful for this. The reason i ask is because there's a few links directly from the start page that require the resigter view to be shown instead of login. Also, you can reuse all the code, just add the <Login/> component the the /register/page.tsx too and make the login form tell which page it's on. The don't have an account button will just need to redirect to /register then